bin/node-pre-gyp

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env node
 
-'use strict';
```

Comment:
Why did you remove this? Would be ideal to leave this in across all the JS files.

Reply:
i remove it because eslint flagged it as unncessary. maybe @mapbox/eslint-config-mapbox/.eslint.rc should have `root: true` set?

Reply:
the rule in my upper-level eslintrc file is:
```
    "strict": [2, "global"],
```
but that seems like it should allow a single 'use strict' at the top of files based on [eslint doc](https://eslint.org/docs/rules/strict).

Reply:
Okay. I'm not seeing that locally / or on travis. So, can you bring back the `strict` here and I'll investigate more this week.

Reply:
i think you're not seeing it because you don't have a upper-level directory with an eslintrc file that supplies defaults for settings not specified by your eslintrc files. i think if you add `root: true` (or `"root": true`) to your file then it won't pick up mine.

[eslint doc](https://eslint.org/docs/user-guide/configuring) - search for root: true.

Reply:
i in-place edited `node_modules/@mapbox/eslinit-config-mapbox/base.js` to start with:

```
module.exports = {
  extends: 'eslint:recommended',
  root: true,
  env: {
    es6: true
  },
```

and restarted vscode. eslint no longer complains about 'use strict'.

Reply:
Okay, can you please revert your removal of `use strict`?

Reply:
am doing so now. will ping you when done.

Reply:
thanks

Reply:
looks like you missed restoring one `use strict`, the one referenced by this comment